gcloud/storage/blob.py

```diff
@@ -149,10 +149,10 @@ def public_url(self):
             bucket_name=self.bucket.name,
             quoted_name=quote(self.name, safe=''))
 
-    def generate_signed_url(self, expiration, method='GET',
-                            client=None, credentials=None,
-                            response_type=None, response_disposition=None,
-                            generation=None):
+    def generate_signed_url(self, expiration, method='GET', client=None,
+                            content_type="application/octet-stream",
```

Comment:
1. Single quotes, okay.
2. It prevents undefined behavior, or to say, explicit is better than implicit. Some Java HTTP clients had no option to exclude content type header.